docs/sources/alerting/unified-alerting/high-availability.md

```diff
@@ -0,0 +1,64 @@
++++
+title = "High Availability"
+description = "High Availability"
+keywords = ["grafana", "alerting", "tutorials", "ha", "high availability"]
+weight = 450
++++
+
+# High Availability - How does it work?
+
+The Grafana alerting systems has two main components, the `Scheduler` and the internal `Alertmanager`. The `Scheduler` is responsible for the evaluation of your [alert rules]({{< relref "/fundamentals/evaluate-grafana-alerts.md" >}}) while the internal Alertmanager takes care of the **routing** and **grouping**.
+
+When it comes to a highly available setup, the operational mode of the scheduler is unaffected. All alerts will be evaluated on every instance. The operational change happens in the Alertmanager, it is in charge of also **deduplicating** the alert notifications between Grafana instances.
+
+```
+  .─────.
+ ╱       ╲                                                                      ┌────────────────┐
+(  User   )──────┐                        ┌──────────────────────────────────┐  │                │
+ `.     ,'       │                        │┌─────────┐      ┌──────────────┐ │  │                ▼
+   `───'         │                        ││Scheduler│──────▶Alertmananager│─┼──┘    ┌──────────────────────┐
+                 │      ┌───────────┐  ┌─▶│└─────────┘      ▲──────────────┤ │       │                      │
+  .─────.        │      │   Load    │  │  │Grafana          │              │ │       │                      │
+ ╱       ╲       │      │ Balancing │  │  └─────────────────┼──────────────┼─┘       │     Integrations     │
+(  User   )──────┼─────▶│  Reverse  │──┤  ┌─────────────────┼──────────────┼─┐       │                      │
+ `.     ,'       │      │   Proxy   │  │  │┌─────────┐      ├──────────────▼ │       │                      │
+   `───'         │      └───────────┘  │  ││Scheduler│──────▶Alertmananager│─┼──┐    └──────────────────────┘
+                 │                     └─▶│└─────────┘      └──────────────┘ │  │                ▲
+  .─────.        │                        │Grafana                           │  │                │
+ ╱       ╲       │                        └──────────────────────────────────┘  └────────────────┘
+(  User   )──────┘
+ `.     ,'
+   `───'
+```
```

Comment:
I wonder if we can use [plant UML](https://plantuml.com/). It lets you write diagrams as a code.

Reply:
I've always used [mermaid](https://mermaid-js.github.io/mermaid-live-editor/edit/?gist=https://gist.github.com/sidharthv96/6268a23e673a533dcb198f241fd7012a#eyJjb2RlIjoiZ3JhcGggVERcbiAgICBBW1BhcnR5XSAtLT58R2V0IG1vbmV5fCBCKEdvIHNob3BwaW5nISEpXG4gICAgIiwibWVybWFpZCI6IntcbiAgXCJ0aGVtZVwiOiBcImZvcmVzdFwiLFxuICBcInRlc3RcIjogXCJoZWxsbyB3b3JsZFwiXG59IiwidXBkYXRlRWRpdG9yIjpmYWxzZSwiYXV0b1N5bmMiOnRydWUsInVwZGF0ZURpYWdyYW0iOnRydWUsImxvYWRlciI6eyJ0eXBlIjoiZ2lzdCIsImNvbmZpZyI6eyJ1cmwiOiJodHRwczovL2dpc3QuZ2l0aHViLmNvbS9zaWRoYXJ0aHY5Ni82MjY4YTIzZTY3M2E1MzNkY2IxOThmMjQxZmQ3MDEyYSJ9fX0) for that type of work. But now prefer these type of diagrams because they're just plain text.

Reply:
For [example](http://www.plantuml.com/plantuml/uml/XP31QiCm38RlVWgHIqyzs0zGy2uXi45OoAqNprQJG6eLHnqTO-y-sOPkR8Vw9Ed__Nz8EwOBCC4RCTCPmfBGyq9HxLOySLRICityIr3ECoLenWtJre3RODJAq_b24ID3yqmkm9DRNVHBd8xn5T_5YD0dkdqqO0VebHerxDF-a4l5D27FcE1JHGe8RN8d5rrERhKCGTvF_k1d3DTLxA8F-ezgHU0N-JrLWws6GN27t7of_mppW64UCQfXC8LXCiCABzJEolXwX_TrxLRVAwFaUEjQZxj1qqN6ARdIbUjsDIwRkv9Mgcy0) 
![image](http://www.plantuml.com/plantuml/png/XP31QiCm38RlVWgHIqyzs0zGy2uXi45OoAqNprQJG6eLHnqTO-y-sOPkR8Vw9Ed__Nz8EwOBCC4RCTCPmfBGyq9HxLOySLRICityIr3ECoLenWtJre3RODJAq_b24ID3yqmkm9DRNVHBd8xn5T_5YD0dkdqqO0VebHerxDF-a4l5D27FcE1JHGe8RN8d5rrERhKCGTvF_k1d3DTLxA8F-ezgHU0N-JrLWws6GN27t7of_mppW64UCQfXC8LXCiCABzJEolXwX_TrxLRVAwFaUEjQZxj1qqN6ARdIbUjsDIwRkv9Mgcy0)

Reply:
Before this decision, I looked at https://grafana.com/docs/grafana/latest/administration/set-up-for-high-availability/ and noticed we tend to do custom graphics for our documentation. 

So I opted out from doing something like the above and instead relied on something that's plain text. I have tagged someone from the docs team to let me know what's the best course of action.

But is very likely that I'll keep the plain text version if you don't mind 😄